src/doc/tutorials/PythonExpressions.rst

```diff
@@ -0,0 +1,202 @@
+.. _Tutorials_Python_Expressions:
+
+Python Expressions
+==================
+
+.. toctree::
+       :maxdepth: 2
+
+This tutorial describes using Python Expressions.
+
+Python expressions overview
+---------------------------
+
+VisIt_ contains the ability to create expressions using the Python scripting language.
+Python expressions run in the compute engine and run in parallel when VisIt_ is run in parallel.
+Python expressions operate on vtkDataSets, which provide access to variables as well as the mesh coordinates and topology.
+Python expressions also have access to VisIt_'s metadata as well as access to MPI when running in parallel.
+Python expressions return a vtkDataArray, which allows returning new variables.
+It isn't possible to change the mesh topology or coordinates.
+The functionality is available through the *GUI* and the *CLI*.
```

Comment:
Aren't terms like **GUI**, **CLI**, **Expressions** window, etc. supposed to be bolded? I think our docs say that.